_posts/2016/2016-05-25-spring-io.md

```diff
@@ -0,0 +1,38 @@
+
+Additionally, multiple security tools were mentioned or explained, including AST (Application Security Testing) and WAF (Web Application Firewall) tools, which let you find vulnerabilities and protect from design flaws respectively.
+
+# "From imperative to reactive", by Rossen Stoyanchev
+Rossen Stoyanchev ([@rstoya05](http://twitter.com/rstoya05)) is a member of Spring Framework's development team at [Pivotal](https://pivotal.io/). He and his colleague Stephane Maldini ([@smaldini](https://twitter.com/smaldini)) opened the second day with an interesting keynote about reactive programming, which later on led to the talk "From imperative to reactive" by Mr. Stoyanchev alone.
+{% image blog/blog-springio-1.jpg %}
+The main idea behind this talk was to design asynchronous APIs that allow applications to use fewer threads that don't block each other. One tool that helps doing that is [Reactor](http://projectreactor.io/), which is a [Spring](https://spring.io/) library.
+
+# "Understanding microservice performance", by Rob Harrop
+Last but not least, Rob Harrop ([@robertharrop](https://twitter.com/robertharrop)) brought some numbers to the table to talk about microservices' performance, which is a combination of throughput and latency. The latter is quite a popular yet ambiguous term which is not as easy to analyse it might seem. In fact, checking its mean is not enough, or actually meaningless, as stated by Rob Harrop himself:
+
+> > <blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">Mean just means meaningless at <a href="https://twitter.com/spring_io">@spring_io</a> <a href="https://twitter.com/GuillermoTbb/status/733608037425008640">https://twitter.com/GuillermoTbb/status/733608037425008640</a></p>&mdash; GuillermoTbb (@GuillermoTbb) <a href="https://twitter.com/GuillermoTbb/status/733608037425008640">11. Mai 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Mr. Harrop showed a couple of tiny web-apps that included some interesting graphics where you could see how performance changes depending on certain variables.
+
+# Conclusion
+During those two days I had the chance to enjoy a conference full of top-notch industry experts who shared their views and knowledge on Spring, microservices and other related topics. It was a pleasant experience that I hope will be repeated again.
```

Comment:
microservices and other related topics > microservices, and other related topics